lib/matplotlib/axes/_axes.py

```diff
@@ -6777,6 +6777,45 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
                           else "list[Polygon]")
             return tops, bins, cbook.silent_list(patch_type, patches)
 
+    def vector(self, dx, dy, x=0., y=0., **kwargs):
+        """
+        Plot an arrow indicating a vector.
+
+        Parameters
+        ----------
+        dx : float
+        The x-component of the vector
+        dy : float
+        The y-component of the vector
+        x : float (optional)
+        The x-coordinate of the vector's base if nonzero.
+        y : float (optional)
+        The y-coordinate of the vector's base if nonzero.
```

Comment:
@kmdalton The description lines need to be indented I think

Reply:
Oops -- I missed that when i was fighting with `flake8` about tabs vs spaces. Should be fixed now.